app/build.gradle

```diff
@@ -93,9 +93,15 @@ dependencies {
     debugImplementation 'androidx.fragment:fragment-testing:1.3.0-rc01'
 
 
+    // RecyclerView for AndroidX
+    implementation "androidx.recyclerview:recyclerview:1.1.0"
+    // For control over item selection of both touch and mouse driven selection
+    implementation "androidx.recyclerview:recyclerview-selection:1.1.0-rc03"
```

Comment:
Should move this up where all other `implementation` lines are, just for organization.